src/validation/RegistrationSchema.tsx

```diff
@@ -31,6 +31,13 @@ export const RegistrationSchema = object().shape({
         PhoneRegex,
         "Please enter a valid phone number, that has no spaces, and includes country code.",
       ),
+    description: string()
+      .label("Business description")
+      .required()
+      .matches(
+        /[a-zA-Z,.?!']/,
+        "Please remove any numbers or special characters.",
```

Comment:
This seems harsh?  "Providing services to FTSE-100 companies" seems a perfectly fine description to me. What are we trying to protect against here?

Reply:
I think my intent is that the description be as sanitized as possible, but maybe being a bit aggressive